deployments/k8s/helm/signalfx-agent/restricted-sfx-agent.yaml

```diff
@@ -0,0 +1,81 @@
+# This Pod Security Policy is setup to allow the signalfx-agent daemonset to
```

Comment:
These files have to be wrapped in a condition to prevent them from being deployed when the `pspMode` value is not set to `restricted`.